content/docs/rfcs/32/README.md

```diff
@@ -0,0 +1,560 @@
+---
+slug: 32
+title: 32/RLN
+name: Rate Limit Nullifier specification
+status: raw
+editor: Blagoj Dimovski <blagoj.dimovski@yandex.com>
+contributors: 
+---
+
+# Abstract
+
+The following document covers the specification of the RLN construct as well as the auxiliary libraries necessary for interaction with it.
+
+# Motivation
+
+Rate limiting nullified (RLN) is a construct based on zero-knowledge proofs that provides an anonymous rate-limited signaling/messaging framework suitable for decentralized (and centralized) environments. 
+Anonymity refers to the unlinkability of messages to their owner.
+RLN guarantees a messaging rate is enforced cryptographically while preserving the anonymity of the message owners.
+A wide range of applications can benefit from RLN and provide desirable security features. 
+For example, an e-voting system can integrate RLN to contain the voting rate while protecting the voters-vote unlinkability. 
+Another use case is to protect an anonymous messaging system against DDoS and spam attacks by containing messaging rate of users. 
+This latter use case is explained in [17/Waku-RLN-Relay RFC](/spec/17).
+
+
+# Flow
+
+The users participate in the protocol by first registering to an application-defined group referred by the _membership group_.
+Registration to the group is mandatory for signaling in the application. 
+After registration, group members can generate Zero-knowledge Proof of membership for their signals and can participate in the application. 
+Usually, the membership requires a financial or social stake which 
+is beneficial for the prevention of Sybil attacks and double-signaling.
+Group members are allowed to send one signal per epoch.
```

Comment:
Since the spec is not gonna be focused on spam protection use case, then I think the `epoch` should be replaced with `external nullifier` throughout the spec.